docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss

```diff
@@ -0,0 +1,136 @@
+.hb-hero-overlay {
+  position: relative;
+  margin: 6% 0 20%;
+
+  // Offset background color psuedo element
+  &::before {
+    @include psuedo-background-box(lighten($su-color-driftwood, 25%), 105%, 85%);
+    top: -5%;
+    left: 0;
+    z-index: $hb-index-pseudo-background;
+  }
+
+  @include grid-media-min('md') {
+    margin: 10% 0 0;
+
+    &::before {
+      top: -10%;
+      height: 90%;
+    }
+  }
+
+  &__image-wrapper {
+    @include clearfix;
+
+    img {
+      width: 100%;
+
+      @include grid-media-min('md') {
+        float: right;
+        width: 80%;
+      }
+    }
+  }
+
+  &__text {
+    background-color: darken($su-color-driftwood, 10%);
+    color: $hb-color--white;
+    display: block;
+    margin-left: 6%;
+    padding: hb-calculate-rems(64px) hb-calculate-rems($hb-gutter-width) hb-calculate-rems($hb-gutter-width);
+    position: relative;
+    transform: translateY(-15%);
+
+    @include hb-colorful {
+      background-color: adjust-color(hb-colorful-variation(primary), $lightness: -9%, $alpha: -0.05);
+    }
+
+    @include grid-media-min('md') {
+      min-width: $hb-hero-width;
+      width: 60%;
+      transform: translateY(-60%);
+    }
+
+    @include grid-media-min('lg') {
+      // 1.3x larger
+      min-width: $hb-hero-width * 1.3;
+      transform: translateY(-70%);
+    }
+
+    h2 {
+      @include hb-heading-4;
+    }
+
+    @include hb-colorful {
+      // Color block psuedo element
+      &::before {
+        top: -35%;
+        left: -10%;
+        @include psuedo-background-box(hb-colorful-variation(secondary), 35%, 10%);
+
+        @include grid-media-min('md') {
+          top: -40%;
+          height: 40%;
+        }
+
+        @include grid-media-min('lg') {
+          top: -35%;
+          height: 35%;
+        }
+      }
+
+      // Line decoration pseudo element
+      &::after {
+        display: block;
+        top: hb-calculate-rems(56px);
+
+        @include psuedo-background-box(
+          lighten(hb-colorful-variation(primary), 30%),
+          hb-calculate-rems(4px),
+          hb-calculate-rems(65px)
+        );
+      }
+    }
+
+    .text-long,
+    a {
+      font-size: hb-calculate-rems(16px);
+
+      @include grid-media-min('md') {
+        font-size: hb-calculate-rems(18px);
+      }
+    }
+
+    a {
+      display: block;
+      // TODO: make this into a border button mixin?
+      border-radius: hb-calculate-rems(42px);
+      font-weight: hb-theme-font-weight(semibold);
+      margin-top: hb-calculate-rems($hb-gutter-width);
+      max-width: hb-calculate-rems(184px);
+      padding: hb-calculate-rems(10px);
+      text-decoration: none;
+      text-align: center;
+
+      @include hb-colorful {
+        border: 3px solid lighten(hb-colorful-variation(primary), 30%);
+        color: lighten(hb-colorful-variation(primary), 30%);
+
+        &:hover,
+        &:focus {
+          box-shadow: none;
+          // what's the hover effect here?
```

Comment:
We'll need to figure this out before merging. The text color darkening is not accessible.

Reply:
Merani added designs that I implemented